File: Day 14/Demo_RC/app/src/main/java/com/example/demo_rc/MainActivity.java
package com.example.demo_rc;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {
    RecyclerView recyclerView_one;

    String name [] = {"mayur","mayur","mayur","mayur","mayur","mayur","mayur","mayur","mayur","mayur",};
    int [] images = {R.drawable.a,R.drawable.c,R.drawable.d,R.drawable.e,R.drawable.f,R.drawable.g,R.drawable.h,R.drawable.a,R.drawable.d,R.drawable.f};
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        recyclerView_one = findViewById(R.id.recyclerView);

        MyAdapter myAdapter = new MyAdapter(images,name);

        recyclerView_one.setLayoutManager(new LinearLayoutManager(this));

        recyclerView_one.setAdapter(myAdapter);


    }
}

File: Siddhi mam Tasks/Day 37/Co-routines/ViewModel_Factory/app/src/main/java/com/example/viewmodel_factory/MainActivity.kt
package com.example.viewmodel_factory

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.lifecycle.ViewModelProvider

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val viewModelFactory = MyViewModelFactory("Mayur")



        var viewModel = ViewModelProvider(this,viewModelFactory).get(MyViewModel::class.java)

    }
}